awx/main/access.py

```diff
@@ -994,7 +998,7 @@ def can_change(self, obj, data):
             self.check_license(add_host_name=data['name'])
 
             # Check the per-org limit
-            self.check_org_host_limit({'inventory': obj.inventory}, add_host_name=data['name'])
+            self.check_org_host_limit({'inventory': obj.inventory}, add_host_name=data['name'], action='change')
```

Comment:
Since `check_org_host_limit` is short-circuiting out of doing any real work when using this kwarg by your changes, wouldn't it make sense to just remove this call entirely and not do the `action == 'change'` short-circuit, unless there is another usage I haven't gotten to yet?